Assets/Scripts/Game/Entities/EnemyEntity.cs

```diff
@@ -107,6 +107,9 @@ public void SetEnemyCareer(MobileEnemy mobileEnemy, EntityTypes entityType)
                 return;
             }
 
+            if (careerIndex == (int)MobileTypes.Knight_CityWatch)
+                level += UnityEngine.Random.Range(3, 7);
+
             this.mobileEnemy = mobileEnemy;
```

Comment:
Careless me, I just realized I applied the additional levels here _after_ health is rolled (although before skill levels are calculated). I'll double-check whether the guards in classic get a health boost from their additional levels.